ideal_weight.rb

```diff
@@ -0,0 +1,35 @@
+# calculate ideal weight
+
+IDEAL_WEIGHT = 110
+
+class Human
+  def initialize(name, height)
+    @name = name
+    @height = height
+  end
+
+  def ideal_weight(name, height)
```

Comment:
you should use variables defined in the constructor. I think, it does not work

Reply:
I deleted this variables in sublime text i did't  know why this is seems here, i will check my code tomorrow